packages/dates/src/utilities/formatDate.ts

```diff
@@ -58,7 +60,12 @@ export function formatDate(
     }).format(adjustedDate);
   }
 
-  return memoizedGetDateTimeFormat(locales, options).format(date);
+  return memoizedGetDateTimeFormat(locales, {
+    ...options,
+    timeZone: options.timeZone
+      ? mapDeprecatedTimezones(options.timeZone)
```

Comment:
HIII @jesstelford ❤️ SOOOOOOOOOO when we implemented the change inside `i18n` and tested out the snapshots ... we weren't able to see the changes actually fix the issue 😬 .... so we're trying it out directly in formatDate as parts of web directly call `formatDate` from `@shopify/dates` we shipped https://github.com/Shopify/web/pull/152962/files that actually fixes it in web, but if we wanted to do our "due diligence" for other repo's, thought we might as well try this LOL 😄

Reply:
Interesting! During your testing, did you notice any _regressions_ with this change (trying some different mapped and non-mapped timezones)? If not, I agree we should ship this as a bug fix to ensure we've covered all our bases.

Reply:
HIII JESS!!! do you mean for the `i18n` testing? Or this current one? SADLY Npm hasn't finished propogating lol, so I'm just waiting to test 😢